### PR TITLE
v4.0.4-rc3 images updation

### PR DESCRIPTION
**Pull request by girichinna27**
[OP-20403](https://devopsmx.atlassian.net/browse/OP-20403)
Update helm with 4.0.4 images

[OP-20403]: https://devopsmx.atlassian.net/browse/OP-20403?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: charts/oes/values.yaml [hunk 1]
```diff
@@ -447,6 +447,14 @@ stormdriver:
   image:
     repository: quay.io/opsmxpublic/stormdriver
     tag: v1.2.6
+####################################################
```
Comment (saitejaopsmx) [inline charts/oes/values.yaml:450]: Revert this changes.. few more files need to be updated

**Review by saitejaopsmx — APPROVED** [review]

**Comment by saitejaopsmx** [comment]
@girichinna27 please check the comments

File: charts/oes/config/oes-visibility/visibility-local.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
+message-broker:
+  enabled: false
+  username: rabbitmq
```
Comment (saitejaopsmx) [inline charts/oes/config/oes-visibility/visibility-local.yml:3]: Need to be picked from values.yaml file..
We should not hard code this.

**Review by saitejaopsmx — COMMENTED** [review]
Please check the comments @girichinna27

File: charts/oes/values.yaml (hunk 1 above)
Comment (girichinna27) [reply charts/oes/values.yaml]: Removed the create controller secret block of code. So please take care of it.

File: charts/oes/config/oes-visibility/visibility-local.yml (hunk 2 above)
Comment (girichinna27) [reply charts/oes/config/oes-visibility/visibility-local.yml]: Updated the visibility-secret file to fetch username/password from values.yaml file

**Review by saitejaopsmx — APPROVED** [review]
